scripts/basic-workflow-tests.sh

```diff
@@ -120,8 +121,8 @@ test_init() {
         ilab config init --non-interactive
     fi
     step Checking config.yaml
-    if [ "${MIXTRAL}" -eq 1 ]; then
-        sed -i -e 's|merlinite.*|mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf|' "${CONFIG_HOME}/instructlab/config.yaml"
+    if [ "${MIXTRAL}" -eq 1 ] && [ "${BACKEND}" == "vllm" ]; then
```

Comment:
`set_defaults` seems to guarantee that it's never `!= "vllm"`. Wonder if we have to check it everywhere below.